templates/mongodb-atlas-main.template.yaml

```diff
@@ -184,12 +183,11 @@ Resources:
     Type: AWS::CloudFormation::Stack
     Properties:
       TemplateURL: !Sub
-        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/register-mongodb-atlas-resources.yaml'
+        - 'https://${S3Bucket}.s3.${S3Region}.${AWS::URLSuffix}/${QSS3KeyPrefix}templates/activate-mongodb-atlas-resources.yaml'
```

Comment:
We should rename the resource from `RegisterAtlasResources` to `ActivateAtlasResources` for consistency. The condition should be renamed as well.